docker-for-windows/troubleshoot.md

```diff
@@ -7,18 +7,57 @@ redirect_from:
+and communicate with the Docker for Windows team, use our forums and Knowledge Hub,
+browse and log issues on GitHub, and find workarounds for known problems.
 
-## Getting help
+## Docker Knowledge Hub
 
-There are several ways to get the support you need with Docker for Windows. If you encounter problems not addressed here in the documentation:
+**Looking for help with Docker for Windows?** Check out the [Docker Knowledge
+Hub](http://success.docker.com/q) for knowledge base articles, FAQs, and
+technical support for various subscription levels.
 
-- Refer to the knowledge base articles at the [Docker Success Center](https://success.docker.com/q/).
-- Browse the logs (in `User\AppData\Local\Docker`) by clicking **log file** in the Diagnose & Feedback window.
-- Ask questions on the [Docker for Windows forum](https://forums.docker.com/c/docker-for-windows).
-- Upload diagnostics in the Diagnose & Feedback window. You'll get a unique ID in return. You can then use this ID to submit issues at the [Docker for Windows GitHub repo](https://github.com/docker/for-win/issues).
+## Diagnose problems, send feedack and create GitHub issues
 
-![Diagnose & Feedback with ID](images/diagnostic-id.png){:width="500px"}
+### In-app diagnostics
+If you encounter problems for which you do not find solutions in this
+documentation, on [Docker for Windows issues on
+GitHub](https://github.com/docker/for-win/issues), or the [Docker for Win
+forum](https://forums.docker.com/c/docker-for-windows), we can help you troubleshoot
+the log data.
+
+Choose ![whale menu](images/whale-x.png){: .inline} →
+**Diagnose & Feedback** from the menu bar.
+
+![Diagnose & Feedback](images/diagnose-feedback.png){:width="600px"}
+
+Once the **Diagnose & Feedback** window is opened, it will start to collect the dignostics. When the diagnostics are available, you can upload them and obtain a **Diagnostic ID**, which must be provided when communicating with the Docker team. For more information on our policy regarding personal data you can read [how is personal data handled in Docker Desktop](https://docs.docker.com/docker-for-mac/faqs/#how-is-personal-data-handled-in-docker-desktop).
+
+![Diagnose & Feedback with ID](images/diagnostic-id.png){:width="600px"}
+
+If you click on **Report an issue**, this opens [Docker for Windows issues on
+GitHub](https://github.com/docker/for-win/issues/) in your web browser in a "create new issue" template, to be completed before submision. Do not forget to copy/paste your diagnistic ID.
+
+![issue-template](images/issue-template.png){:width="600px"}
+
+### Diagnosing from the terminal
+
+On occasions it is useful to run the diagnostics yourself, for instance if Docker for Windows cannot start.
+
+First locate the `com.docker.diagnose`, that should be in `C:\Program Files\Docker\Docker\resources\com.docker.diagnose.exe`.
+
+To create *and upload*  diagnostics in Powershell, run:
+
+```sh
```

Comment:
This should probably be `powershell`, not `sh`

Reply:
Also (I'm not a windows user), is `> &` part of the command to run, or is that the prompt?

Reply:
it didn't seem clear, I've changed it to `PS C:\>` because the `&` is part of the command :D